components/director/pkg/graphql/schema.graphql

```diff
@@ -533,6 +620,31 @@ type OneTimeTokenForRuntime implements OneTimeToken {
 	rawEncoded: String
 }
 
+type PackageDefinition {
+	id: ID!
+	name: String!
+	description: String
+	authRequestJSONSchema: JSONSchema
+	auth(id: ID!): APIInstanceAuth
+	auths: [APIInstanceAuth!]!
+	"""
+	When defined, all Auth requests fallback to defaultAuth.
+	"""
+	defaultAuth: Auth
```

Comment:
if `defaultAuth` applies only to API, maybe it should be called: `apisDefaultAuth`

Reply:
Hmm... okay, but then the `auths` field would need to be called `apisAuths`. It is fine for me, but I feel the naming would be too complicated. apisAuths would suggest that auth is created per single API Definition, not per Package. 

I would keep `defaultAuth` and `auths`, as they are on Package level.